common/utilities/tests/test_config.py

```diff
@@ -0,0 +1,26 @@
+import unittest
+from unittest.mock import patch
+
+import utilities.config as config
+
+
+@patch("utilities.config.config", new_callable=dict)
```

Comment:
No major issues with the approach here, but it looks like [`patch.dict`](https://docs.python.org/3/library/unittest.mock.html#patch-dict) would also be an option. It might make things a little more readable?